learn/experimental/autobatching.md

```diff
@@ -0,0 +1,34 @@
+# Auto-batching
+
+If enabled, this feature automatically combines multiple document addition requests ([`POST`](/reference/api/documents.md#add-or-replace-documents) or [`PUT`](/reference/api/documents.md#add-or-update-documents)) into batches to speed up indexation.
+
+## What does "experimental" mean?
+
+This is an experimental feature added in `v0.26` of Meilisearch, meaning that it has not been tested to the same standards as our other features and may be reworked or removed in a future Meilisearch release.
+
+We would like your feedback! [Join the discussion](https://github.com/meilisearch/meilisearch/discussions/2070) (GitHub account required).
+
+## Enable auto-batching
+
+To enable auto-batching, start Meilisearch while supplying the `--enable-autobatching` CLI flag:
+
+```
+./meilisearch --enable-autobatching
+```
+
+## Customize auto-batching behavior
+
+There are three options for customizing auto-batching behavior:
+
+- `debounce-duration-sec`: the number of seconds to wait between receiving a document addition task and beginning the batching and indexation process. **Default: `0`**
```

Comment:
I think we can add this part here as well:

From the [github discussion](https://github.com/meilisearch/meilisearch/discussions/2070):

>  This feature may be useful when you want to send large payloads, but don't want to increase the maximum payload size for the server, you can instead split the payload, send each chunk, and given that you have a debounce duration set high enough, all the chunks will be processed together.

Reply:
I will leave this off for now since it seems like a niche case.